internal/core/note_index.go

```diff
@@ -150,7 +150,8 @@ func (t *indexTask) execute(callback func(change paths.DiffChange)) (NoteIndexin
 		return false, nil
 	}
 
-	source := paths.Walk(t.path, t.logger, shouldIgnorePath)
+    notebookPath := &NotebookPath{Path: t.path}
```

Comment:
Please mind whitespaces. You're using spaces whereas in the file tabs are used.

Reply:
this is really weird. mcDevnagh picked up on the same issue from another PR. But I'm using gopls to format. Both from nvim via formatter.nvim and from the cli, gopls formats for me like this. 

not sure what to do about thath?

Reply:
ups! forgot to run the formatter 😅
Sharp eyes!